tools/Modules/TestFx-Tasks.psm1

```diff
@@ -1,4 +1,167 @@
 ﻿[CmdletBinding]
+
+Function New-AzureCredential
+{
+    [CmdletBinding(
```

Comment:
What are we doing to test this?  I worry that this could break in the future without us knowing.  How should we test?

Reply:
Done. Also updated documentation.